examples/shared-modules/README.md

```diff
@@ -0,0 +1,22 @@
+# Example app using shared modules
+
+This example features:
+
+* An app with two pages which has a Counter component
+* That Counter component maintain the counter inside it's module.
+
+## How to run it
+
+Install dependencies on main project and build dependencies.
```

Comment:
This README is focused on running from the source. When we launched this version, we can change the README to just do:

```sh
npm i
npm run dev
```